src/main/java/utils/parser/DeckKeywordParser.java

```diff
@@ -28,8 +31,8 @@ private static Options buildDeleteOptions() {
         Options options = new Options();
         options.addRequiredOption("d", "deck", true, "deck name");
         // extra stuff down here
-        //        options.addOption("c", "card", true, "card name");
-        //        options.addOption("t", "tag", true, "tag name");
+        options.addOption("c", "card", true, "card name");
```

Comment:
Ah, I'll make the changes and push then

Reply:
`private static Options buildListOptions() {
        Options options = new Options();
        options.addOption("c", "cards", true, "deck name");
        options.addOption("t", "tags", true, "deck name");
        return options;
    }`
    
    I think this might still be an issue, in spite of adding `(optional)`.

Reply:
Have modified the relevant parts of the code and pushed them through!